posthog/test/base.py

```diff
@@ -199,6 +200,9 @@ def setUp(self):
 
         # Clear the cached "is_cloud" setting so that it's recalculated for each test
         TEST_clear_cloud_cache()
+        # Clear the is_rate_limit lru_Caches so that they does not flap in test snapshots
```

Comment:
this is the meat of it, the rest is noise caused by snapshots changing or needing to change expected query counts